frontend/backend/ofrak_server.Dockerfile

```diff
@@ -1,12 +1,11 @@
 FROM redballoonsecurity/ofrak/dev:latest
 
-COPY ofrak_server.py /ofrak_server.py
-COPY docs /docs
-COPY mkdocs.yml /mkdocs.yml
```

Comment:
Why remove the docs from the development Dockerfile?

Reply:
Moreover, if you remove the docs from the image, the `mkdocs serve` run in the entrypoint probably won't work?

Reply:
Clicking on a tag in the frontend takes me to the public ofrak.com docs anyway, so the dev server hosted docs are basically unused. If we were testing the documentation build here, we should have a dedicated server build for that (in the `docs` directory).

Reply:
And you're right about the mkdocs serve commend, I mean to remove that and could have sworn I had. Definitely removed now.